src/pylorax/ltmpl.py

```diff
@@ -553,15 +559,15 @@ def removefrom(self, pkg, *globs):
                 logger.debug("removefrom %s %s: no files matched!", pkg, g)
         # are we removing the matches, or keeping only the matches?
         if keepmatches:
-            remove = filelist.difference(matches)
```

Comment:
I'd rather just rename this instead of using _remove, maybe remove_files or remove_list